packages/e2e-test-utils-playwright/src/page/press-key-with-modifier.ts

```diff
@@ -0,0 +1,138 @@
+/**
+ * External dependencies
+ */
+import { capitalize } from 'lodash';
+import type { Page } from '@playwright/test';
+
+/**
+ * WordPress dependencies
+ */
+import { modifiers, SHIFT, ALT, CTRL } from '@wordpress/keycodes';
+import type { WPKeycodeModifier } from '@wordpress/keycodes';
+
+/**
+ * Internal dependencies
+ */
+import type { PageUtils } from './index';
+
+let clipboardDataHolder: {
```

Comment:
Why do we need this object that is used and is updated in the helper functions?

Reply:
This is a refactor from saving it in the `window._clipboardData` (which pollutes the global scope) to saving it using a [JSHandle](https://playwright.dev/docs/api/class-jshandle).

I tested it in the other tests migrated here and it seems to work fine. However, there might be edge cases I missed that break the new tests though. If you've spotted any inconsistency or error, please LMK :)!

Reply:
This potentially (not tested 😅) also fixes a bug in the original util where copied data won't persist after page navigations (Because `window._clipboardData` will be destroyed). 

_(But now that I think about it, using local module scope might be problematic if we want to run tests in parallel someday. Maybe we should bind it under the `pageUtils` class scope 🤔. But that's for another day I guess.)_